### PR TITLE
Pop-up-menu feature

### PR DESCRIPTION
**Pull request by Anny85-code**
### In this milestone I;
 - [x] Created the pop-up menu and render it dynamically with JavaScript.
 - [x] Styled it with CSS.
 - [x] deployed.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,36 +1,40 @@
 <!DOCTYPE html>
```
Comment (VuDej) [inline dist/index.html:1]: giphy-downsized
Good job so far! 
But there are still some changes you should make.

File: src/modules/pop-up.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const popUpContainer = document.querySelector('.pop-up-container');
```
Comment (VuDej) [inline src/modules/pop-up.js:1]: It was really nice you created pop-up but it should display image, title and info element clikced on. Kindly use API to get data. You should use id of elements to display data for it. Once you made changes, send againt review request.

**Review by VuDej — CHANGES_REQUESTED** [review]
Required changes

File: src/modules/pop-up.js (hunk 2 above)
Comment (Anny85-code) [reply src/modules/pop-up.js:1]: Thank you for your review. changes will be implemented.

**Pull request merged**